doc/developers-guide/Testing-MongooseIM.md

```diff
@@ -2,40 +2,30 @@
 
 The test runner script is used to compile MongooseIM and run tests.
 
-## IMPORTANT!
-
-* ODBC preset can only be tested [on Ubuntu Xenial x64](../operation-and-maintenance/known-issues.md).
-* SELinux may prevent containers from accessing the disk. Please either disable it or add proper rules to the policy.
-
 ## Requirements
 
 ### Docker
 
 Docker must be installed on the local system and the user executing the tests must have privileges to start new containers (usually achieved by adding the user to the `docker` group).
 
-### MSSQL connectivity
+### FreeTDS for MSSQL connectivity
 
 MongooseIM requires FreeTDS in order to connect to MSSQL container.
 
-First of all, please install the driver itself:
+Please install the driver:
 
 ```bash
 # Ubuntu
 $ sudo apt install freetds-dev tdsodbc
 
-# CentOS
```

Comment:
Why remove this?

Reply:
My thinking was that the `Driver` and `Setup` variables (mentioned in the following lines) are set correctly for Ubuntu/macOS in the scripts. I was considering removing CentOS mentions altogether. But as you pointed out, I think there is no reason for removing this, I will revert this change.